bayesian/api_v1.py

```diff
@@ -281,6 +281,9 @@ class StackAnalysesByGraphGET(ResourceWithSchema):
 
     @staticmethod
     def get(external_request_id):
+        if (get_request_count(rdb, external_request_id) < 1):
```

Comment:
@fridex I don't know why the C programmer in me catches me unawares at times :-(. Changed. Next commit en route.